forge-gui/res/cardsfolder/rebalanced/a-circle_of_the_land_druid.txt

```diff
@@ -0,0 +1,10 @@
+Name:A-Circle of the Land Druid
+ManaCost:1 G
+Types:Creature Gnome Druid
+PT:2/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigMill | TriggerDescription$ When CARDNAME enters the battlefield, you may mill four cards.
+SVar:TrigMill:DB$ Mill | NumCards$ 4 | Defined$ You | Optional$ True
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Graveyard | ValidCard$ Card.Self | Execute$ TrigChangeZone | TriggerDescription$ Natural Recovery — When CARDNAME dies, return target land card from your graveyard to your hand.
+SVar:TrigChangeZone:DB$ ChangeZone | Origin$ Graveyard | Destination$ Hand | ValidTgts$ Land.YouCtrl
```

Comment:
`ValidTgts$ Land.YouOwn | TgtPrompt$ Select target land card from your graveyard`
also the non `A-` version – sorry :(